client/containers/view-profile.js

```diff
@@ -4,13 +4,13 @@ export default class ViewProfile extends Component {
   render() {
     const { displayName, firstName, lastName, city, country, imageUrl, bio, facebook, twitter, instagram, soundcloud } = this.props.user
     return (
-      <div>
-        <div className="jumbotron container">
-          <img className="img-fluid profile-image float-md-left" src={imageUrl}/>
-          <h3 className="display-4">{displayName}</h3>
-          <p className="lead">{firstName + ' ' + lastName}</p>
-          <p className="lead">{city + ', ' + country}</p>
-          <div className="lead">
+      <div className="container">
```

Comment:
`reactstrap` has layout components for `Container`, `Row`, etc.
https://reactstrap.github.io/components/layout/